### PR TITLE
rg-94: not allow register with email with one symbol

### PR DESCRIPTION
**Pull request by ValentinaTenus**

### CODE REVIEW

**Review by Vitaliy-Kalenichenko — APPROVED** [review]

File: shared/src/bundles/users/enums/user-validation-message.enum.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ enum UserValidationMessage {
     LASTNAME_REQUIRE = 'Last name is required',
     EMAIL_REQUIRE = 'Email is required',
     EMAIL_WRONG = 'Email is wrong',
+    EMAIL_INVALID = 'Invalid email format. Please use a valid email address',
```
Comment (jurok3x) [inline shared/src/bundles/users/enums/user-validation-message.enum.ts:6]: what's difference between EMAIL_WRONG and EMAIL_INVALID? What purpose of the first one?

Comment (ValentinaTenus) [reply shared/src/bundles/users/enums/user-validation-message.enum.ts:6]: email invalid for custom validation error. wrong - if it doesn't have email format

Comment (jurok3x) [reply shared/src/bundles/users/enums/user-validation-message.enum.ts:6]: as I see from the message EMAIL_INVALID tells user about **Invalid email format.** so why do we need both?
can you provide real example when user get each of them?

Comment (ValentinaTenus) [reply shared/src/bundles/users/enums/user-validation-message.enum.ts:6]: see specification and MY changes.

**Review by jurok3x — APPROVED** [review]

**Pull request merged**